Makefile

```diff
@@ -50,9 +50,21 @@ endif
 
 # Set testing path for tests-modules
 ifeq ($(FOCUS),"")
-       TESTS_PATH = ./modules/... ./global-hooks/... ./ee/modules/... ./ee/fe/modules/...
+	TESTS_PATH = ./modules/... ./global-hooks/... ./ee/modules/... ./ee/fe/modules/...
 else
-       TESTS_PATH = $(wildcard ./modules/*-${FOCUS} ./ee/modules/*-${FOCUS} ./ee/fe/modules/*-${FOCUS})/...
+	CE_MODULES = $(shell find ./modules/ -maxdepth 1 -regextype sed -regex ".*[0-9]\{3\}-${FOCUS}")
```

Comment:
shell find ./modules/ -maxdepth 1 -regextype sed -regex ".*[0-9]\{3\}-${FOCUS}" cannot work on mac

Reply:
:(
```
find: -regextype: unknown primary or operator
```

Reply:
I've addressed this issue in https://github.com/deckhouse/deckhouse/pull/3740/commits/ecf000f89a99d840a6578e65e2d17c59a0bcb194.